spec-tests/src/test/java/org/candlepin/spec/jobs/JobScheduleSpecTest.java

```diff
@@ -0,0 +1,137 @@
+import static org.candlepin.spec.bootstrap.assertions.StatusCodeAssertions.assertNotFound;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import org.candlepin.ApiException;
+import org.candlepin.dto.api.v1.AsyncJobStatusDTO;
+import org.candlepin.dto.api.v1.OwnerDTO;
+import org.candlepin.dto.api.v1.PoolDTO;
+import org.candlepin.dto.api.v1.ProductDTO;
+import org.candlepin.resource.OwnerApi;
+import org.candlepin.resource.PoolsApi;
+import org.candlepin.spec.bootstrap.client.ApiClient;
+import org.candlepin.spec.bootstrap.client.ApiClients;
+import org.candlepin.spec.bootstrap.client.SpecTest;
+import org.candlepin.spec.bootstrap.client.api.JobsClient;
+import org.candlepin.spec.bootstrap.data.builder.Owners;
+import org.candlepin.spec.bootstrap.data.util.StringUtil;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.time.temporal.ChronoUnit;
+
+@SpecTest
+public class JobScheduleSpecTest {
+
+    private static ApiClient client;
+    private static JobsClient jobsClient;
+
+    @BeforeAll
+    public void beforeAll() throws ApiException {
+        client = ApiClients.admin();
+        jobsClient = client.jobs();
+    }
+
+    @Test
+    @DisplayName("should not schedule arbitrary tasks")
+    public void shouldNotScheduleArbitraryTasks() throws Exception {
+        assertForbidden(() -> jobsClient.scheduleJob("totally-made-up"));
+    }
+
+    @Test
+    @DisplayName("should not schedule non cron tasks")
+    public void shouldNotScheduleNonCronTasks() throws Exception {
+        assertForbidden(() -> jobsClient.scheduleJob("UndoImportsJob"));
+    }
+
+    @Test
+    @DisplayName("should schedule cron tasks irrespective of the case")
+    public void shouldScheduleCronTasksCaseIndependant() throws Exception {
+        AsyncJobStatusDTO jobStatus = client.jobs().scheduleJob("ExpiredPoolsCleanupJob");
+        assertEquals("CREATED", jobStatus.getState());
+        jobsClient.waitForJobToComplete(jobStatus.getId(), 15000);
+    }
+
+    @Test
+    @DisplayName("should purge expired pools")
+    public void shouldPurgeExpiredPools() throws Exception {
+        OwnerApi ownerApi = client.owners();
+        PoolsApi poolsApi = client.pools();
+        OwnerDTO owner = ownerApi.createOwner(Owners.random());
+        ProductDTO product = new ProductDTO();
+        product.setId(StringUtil.random("ID"));
+        product.setName(StringUtil.random("Test Product"));
+        product = client.ownerProducts().createProductByOwner(owner.getKey(), product);
+
+        PoolDTO pool = new PoolDTO();
+        pool.setProductId(product.getId());
+        // TODO: Setting these on the pool causes issues in Jenkins.
+        // Will put back after pool tests have been implemented
```

Comment:
pool_resource_spec has been merged, so you can add these now; although I'm not sure how that affects the tests here

Reply:
Thinking a bit more on this: why do we even need to set these? It doesn't make a difference for the job spec tests (they already pass). Just setting the quantity should be enough

Reply:
I was reproducing the test as written in Ruby. I will rebase and see if the new pool test branch changes anything.